packages/fortunes.rb

```diff
@@ -0,0 +1,14 @@
+require 'package'
+
+class Fortunes < Package
+  description 'Over 15000 cookies for the fortune program.'
+  homepage 'https://packages.debian.org/sid/fortunes'
+  version '1.99.1'
+  source_url 'http://cdn-fastly.deb.debian.org/debian/pool/main/f/fortune-mod/fortune-mod_1.99.1.orig.tar.gz'
```

Comment:
Debian really should get with the times and use https instead but this is what we're dealt and have to live with.

Reply:
@uberhacker, they have an HTTPS mirror. `https://deb.debian.org/` is the usual one. It redirects to the closest HTTPS mirror.